src/logic-operators-extended.js

```diff
@@ -30,19 +30,16 @@ NOTE: The combining operators (&&, ||, &&!, ||!) have lower precedence than
 'use strict';
 import bitsy from "bitsy";
 import {
-	kitsyInit
+	inject
 } from "./kitsy-script-toolkit.js";
 
-var kitsy = kitsyInit();
-
-
-kitsy.inject('operatorMap.set("-", subExp);',
+inject('operatorMap.set("-", subExp);',
 	'operatorMap.set("!==", notEqExp);',
 	'operatorMap.set("&&", andExp);',
 	'operatorMap.set("||", orExp);',
 	'operatorMap.set("&&!", andNotExp);',
 	'operatorMap.set("||!", orNotExp);');
-kitsy.inject('var operatorSymbols = ["-", "+", "/", "*", "<=", ">=", "<", ">", "=="];',
+inject('var operatorSymbols = ["-", "+", "/", "*", "<=", ">=", "<", ">", "=="];',
 	'operatorSymbols.unshift("!==", "&&", "||", "&&!", "||!");');
```

Comment:
While you're here, `!==` should be moved to the end of the new operator list. I didn't really understand that Bitsy's order of operations starts at the end of the list, and `!==` should be more tightly coupled to its left and right than `&&`, `||`, etc.